build/InstallPowerShellCore.cmd

```diff
@@ -1,5 +0,0 @@
-if not exist "%USERPROFILE%\.dotnet\tools\pwsh.exe" (
-  dotnet tool install --global PowerShell --version 6.2.3
-)
-
-set PowerShellCore="%USERPROFILE%\.dotnet\tools\pwsh.exe"
```

Comment:
Is the pre-installed `pwsh.exe` at a different path in the new container image?

Reply:
It's on the PATH in the new container image. We can treat the file path of `pwsh.exe` as opaque now.

Reply:
I just checked to see if it happened to be at the same or different path. Looks like the pre-installed pwsh.exe is not at `%USERPROFILE%\.dotnet\tools\pwsh.exe`